### DISCUSSION TITLE
Can I retrain a checkpoint with new data?

### DISCUSSION THREAD

**Discussion by robinlaegler** [argosopentech/argos-translate#148] [Q&A]
Is it possible to retrain a checkpoint with new datasets or retrain a checkpoint with smaller batch size?
Or do I need to start the training of my language model from scratch every single time without using the checkpoint I already have?

**Comment by PJ-Finlay** [comment]
With  OpenNMT/PyTorch models you can continue training from a checkpoint but once the models are quantized and exported as Argos Translate packages you probably want to start from scratch.

**Reply by robinlaegler** [reply]
Thanks for your quick answer. 
Is there a command to retrain a checkpoint in the argostranslate Training Script that I can use for my purposes or how can I retrain a OpenNMT/PyTorch checkpoint with new data in general?

**Reply by PJ-Finlay** [reply]
I'm not sure if retraining is worth the trouble, if you run the Argos Translate training scripts from scratch it only takes a day or two to train a model. You'd have to figure out retraining/fine tuning yourself, I don know how to do it.
